Makefile

```diff
@@ -297,6 +297,19 @@ GOPATH=$(shell go env GOPATH)
 
 get-deps: .get-deps-stamp
 
+.generic-rpm-done: pause-container-release
+	cp packaging/generic-rpm/ecs-agent.spec ecs-agent.spec
+	cp packaging/generic-rpm/ecs.service ecs.service
+	cp packaging/generic-rpm/amazon-ecs-volume-plugin.service amazon-ecs-volume-plugin.service
+	cp packaging/generic-rpm/amazon-ecs-volume-plugin.socket amazon-ecs-volume-plugin.socket
+	tar -czf ./sources.tgz agent scripts
+	test -e SOURCES || ln -s . SOURCES
+	rpmbuild --undefine=_disable_source_fetch --define "%_topdir $(PWD)" -bb ecs-agent.spec
```

Comment:
```
--undefine=_disable_source_fetch
```

what does this do?

Reply:
Before, the RPM would never download the CNI tarbells from s3, that line was needed to enable downloading the sources in the .spec file.

Reply:
While following with how bottle rocket is built, the submodules were not used in the .spec file and have been removed from the branch.